frontend/src/pages/TermPlanner/DraggableCourse/DraggableCourse.tsx

```diff
@@ -3,28 +3,31 @@ import { useContextMenu } from 'react-contexify';
 import { useSelector } from 'react-redux';
 import ReactTooltip from 'react-tooltip';
 import { InfoCircleOutlined, WarningOutlined } from '@ant-design/icons';
-import { Typography } from 'antd';
+import { Badge, Typography } from 'antd';
 import { useTheme } from 'styled-components';
 import Spinner from 'components/Spinner';
 import type { RootState } from 'config/store';
 import useMediaQuery from 'hooks/useMediaQuery';
 import ContextMenu from '../ContextMenu';
+import { getNumTerms } from '../utils';
 import S from './styles';
 
 type Props = {
   code: string;
   index: number;
   term: string;
+  isDragged: boolean;
```

Comment:
can we rename this to `showMultiCourseBadge` or smth like that rather than `isDragging` for better semantics?